sdk/src/test/java/com/karhoo/sdk/api/service/quotes/VerifyQuoteInteractorTest.kt

```diff
@@ -0,0 +1,129 @@
+        interactor = VerifyQuoteInteractor(credentialsManager, apiTemplate, context)
+    }
+
+    /**
+     * Given:   A request is made to check fleet coverage of an area
+     * When:    There is coverage in the area
+     * Then:    A successful response is returned
+     **/
+    @Test
+    fun checkVerifyQuoteSuccessResponse() {
+        val quoteInfo = Quote(id, quoteType, quoteSource, price, fleet, null, vehicle, vehicleAttributes)
+        whenever(apiTemplate.verifyQuotes(id))
+                .thenReturn(CompletableDeferred(Resource.Success(quoteInfo)))
+        interactor.quoteIdRequest = quoteRequest
+
+        var returnedVerifyInfo: Quote? = null
+        runBlocking {
+            interactor.execute { result ->
+                when (result) {
+                    is Resource.Success -> returnedVerifyInfo = result.data
+                    is Resource.Failure -> Assert.fail()
+                }
+            }
+            delay(5)
+        }
+        Assert.assertNotNull(returnedVerifyInfo)
+        Assert.assertEquals(quoteInfo, returnedVerifyInfo)
+        verify(apiTemplate).verifyQuotes(id)
+    }
+
+    /**
+     * Given:   A request is made to check fleet coverage of an area
+     * When:    There is no coverage in the area
+     * Then:    A failure response is returned
+     **/
+    @Test
+    fun checkVerifyQuoteFailureResponse() {
+        val quoteInfo = Quote(id, quoteType, quoteSource, price, fleet, null, vehicle, vehicleAttributes)
+        whenever(apiTemplate.verifyQuotes(id))
+                .thenReturn(CompletableDeferred(Resource.Success(quoteInfo)))
+        interactor.quoteIdRequest = quoteRequest
+
+        var returnedVerifyInfo: Quote? = null
+        runBlocking {
+            interactor.execute { result ->
+                when (result) {
+                    is Resource.Success -> returnedVerifyInfo = result.data
+                    is Resource.Failure -> Assert.fail()
+                }
+            }
+            delay(5)
+        }
+        Assert.assertNotNull(returnedVerifyInfo)
+        Assert.assertEquals(quoteInfo, returnedVerifyInfo)
+        verify(apiTemplate).verifyQuotes(id)
+    }
+
+    /**
+     * Given:   Id is not set
+     * When:    A request is made to verify quote
+     * Then:    An Internal SDK error is returned
+     **/
+    @Test
+    fun verifyQuoteReturnsAnErrorIfNoIdIsGiven() {
+        var shouldBeNull: Quote? = null
+        var error: KarhooError? = null
+
+        runBlocking {
+            interactor.execute { result ->
+                when (result) {
+                    is Resource.Success -> shouldBeNull = result.data
+                    is Resource.Failure -> error = result.error
+                }
+            }
+            delay(5)
+        }
+
+        Assert.assertEquals(KarhooError.InternalSDKError, error)
+        Assert.assertNull(shouldBeNull)
+    }
+
+    companion object {
+        val quoteType = QuoteType.ESTIMATED
```

Comment:
Should probably be const and names in caps

Reply:
Not sure if it is required to be in Caps and const does not work with this so far. Android does not like the usage of Const on these val's. 

But should be fine to be as it is, taken from another companion object in other interactor tests so doesn't seem to be an issue.

Reply:
https://kotlinlang.org/docs/reference/coding-conventions.html